.github/workflows/build.yml

```
@@ -326,9 +326,23 @@ jobs:
           node-version: ${{ needs.setup-build.outputs.node-version }}
           os: ${{ runner.os }}
 
+      - name: Restore tsbuildinfo cache
+        uses: actions/cache/restore@0057852bfaa89a56745cba8c7296529d2fc39830 # v4.3.0
+        with:
+          path: .cache/tsbuildinfo
+          key: tsbuildinfo-${{ hashFiles('pnpm-lock.yaml') }}
```

```suggestion
          key: tsbuildinfo-${{ hashFiles('pnpm-lock.yaml') }}-${{ github.sha }}
```
Shouldn't this match the save below